experimental/packages/opentelemetry-sdk-metrics-base/src/MeterProvider.ts

```diff
@@ -82,11 +87,11 @@ export class MeterProvider {
     // TODO add a timeout - spec leaves it up the the SDK if this is configurable
     this._shutdown = true;
 
-    // Shut down all exporters and readers.
+    // Shut down all readers.
     // Log all Errors.
-    for (const exporter of this._metricExporters) {
+    for (const collector of this._sharedState.metricCollectors) {
```

Comment:
comment should say collectors not readers no?

Reply:
Yeah, I should update all internal comments to refer to MetricReader with the term MetricCollector.

Reply:
Updated! :)